src/dfcx_scrapi/tools/dataframe_functions.py

```diff
@@ -751,7 +754,8 @@ def bulk_create_intent_from_dataframe(
             self.progress_bar(i, len(temp_intents))
             if update_flag:
                 time.sleep(rate_limiter)
-                self.intents.create_intent(agent_id=agent_id, obj=new_intent)
+                self.intents.create_intent(agent_id=agent_id, obj=new_intent,
```

Comment:
@zya-codes [create_intent](https://github.com/GoogleCloudPlatform/dfcx-scrapi/blob/66b349e2771e84f25f440f28b64a8788e729647a/src/dfcx_scrapi/core/intents.py#L465) doesn't have direct support for a `language_code` arg in its current format, so this won't work.

We'll need to refactor `create_intent` to use the `CreateIntentRequest` similar to how `update_intent` is working today. Once that is complete then we can merge this change in.

Reply:
Actually, I was looking at an outdated ref! 😅 
This change is good to go.